frontend/src/views/Export.vue.js

```
@@ -0,0 +1,33 @@
+import ExportAnnotations from '@/components/VideoExamination/ExportAnnotations.vue';
+debugger; /* PartiallyEnd: #3632/scriptSetup.vue */
```

Biome (2.1.2)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)